src/librustc_mir/util/liveness.rs

```diff
@@ -50,17 +49,13 @@ use util::pretty::{dump_enabled, write_basic_block, write_mir_intro};
 pub type LiveVarSet<V> = IdxSetBuf<V>;
 
 /// This gives the result of the liveness analysis at the boundary of
-/// basic blocks. You can use `simulate_block` to obtain the
-/// intra-block results.
+/// basic blocks.
 ///
 /// The `V` type defines the set of variables that we computed
 /// liveness for. This is often `Local`, in which case we computed
 /// liveness for all variables -- but it can also be some other type,
 /// which indicates a subset of the variables within the graph.
 pub struct LivenessResult<V: Idx> {
```

Comment:
We *could* remove the `V` parameter here and do everything in terms of `Local` again, since this is now just used for the generator transform, and it uses an identity map. But it seems like it's... maybe useful in the future? I dunno.